csunplugged/templates/plugging_it_in/lesson.html

```diff
@@ -0,0 +1,44 @@
+{% extends "base-students.html" %}
+
+{% load i18n %}
+{% load static %}
+{% load render_html_field %}
+{% load django_bootstrap_breadcrumbs %}
+
+{% block title %}
+  {{ topic.name }} Programming Exercises
```

Comment:
```suggestion
  {% blocktrans trimmed %}{{ topic.name }} Programming Exercises{% endblocktrans %}
```

Reply:
Not quite sure how the `{% blocktrans trimmed %}` block works but when this is added in the dynamic content disappears, so I haven't made this change for now. Any ideas on how to fix this?

Reply:
Ah that's my bad. What if you try this: 
```suggestion
  {% blocktrans with name=topic.name %}{{ name }} Programming Exercises{% endblocktrans %}
```

Here is the relevant docs if that still doesn't work: https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#blocktrans-template-tag

Reply:
Cool, I'll give it a try, thanks!

Reply:
Yep, that worked :+1: So did the one below.